awx/main/tests/functional/test_projects.py

```diff
@@ -436,21 +436,21 @@ def test_project_list_ordering_by_name(get, order_by, expected_names, organizati
 
 @pytest.mark.parametrize('order_by', ('name', '-name'))
 @pytest.mark.django_db
-def test_project_list_ordering_with_duplicate_names(get, order_by, organization_factory):
+def test_project_list_ordering_with_duplicate_names(get, order_by, organization_factory, admin):
```

Comment:
does this test even make sense anymore?

Reply:
nvm the test is still valid in ensuring stable return of project list. I think the test title could be updated to remove the "duplicate names" part

Reply:
This is a case where changing the test title wouldn't be completely honest. I undid some stuff that I shouldn't have. I'm going to push a commit that reverts part of this change, and puts it closer to the prior behavior. You can have duplicate names, just not in the same organization, so that's what we should be doing here.